dashboard/test/ui/runner.rb

```diff
@@ -313,7 +320,9 @@ def output_synopsis(output_text)
 
     message += "<br/><i>rerun: ./runner.rb -c #{browser_name} -f #{feature} --html</i>"
     HipChat.log message, color: 'red'
-    HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    if $options.log_dev
+      HipChat.developers short_message, color: 'red' if CDO.hip_chat_logging
+    end
```

Comment:
Ahh yes, from the test machine it will. I do think that if we want to override that behavior it should be opt-in rather than opt-out (i.e. default should be that we log from test environment unless otherwise specified). I'm not sure I'm fully convinced that it's a bad thing to always log when running from test environment, though I don't feel super strongly here.